src/Slide.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <div class="VueCarousel-slide">
+  <div class="VueCarousel-slide" tabindex="-1">
```

Comment:
Hmm, I had never seen this before - really cool. 
If we have form content inside the slider, will the negative `tabindex` prevent the user from focusing, or will a positive tab index in the slide override the negative on the parent 🤔?

Reply:
I use tabindex="-1" for prevent focus from slides, because we can't control slides via keyboard.

Reply:
Yeah, I got that,  just making sure it won't affect content inside the slide because of some weird hierarchy thing. I just looked up the interaction online and it seems fine, so all good here.